src/Sulu/Component/Content/Compat/LocalizationFinder.php

```diff
@@ -71,7 +71,7 @@ public function findAvailableLocale($webspaceName, array $availableLocales, $loc
             return;
         }
 
-        return $resultLocalization->getLocalization();
+        return $resultLocalization->getLocale();
```

Comment:
Just fixed all of these calls because the deprecation messages were annoying during debugging.